src/js/background/background.js

```diff
@@ -90,16 +90,13 @@ async function patchMaskInfo(method = "PATCH", id, data, opts=null) {
 async function storeRuntimeData() {
   // If we already fetched Premium availability in the past seven days,
   // don't fetch it again.
-  const existingPremiumAvailability = (await browser.storage.local.get("premiumCountries")).premiumCountries;
-  const existingIntroPricingEndDate = (await browser.storage.local.get("introPricingEndDate")).introPricingEndDate;
+  const existingPremiumAvailability = (await browser.storage.local.get("periodicalPremiumPlans")).periodicalPremiumPlans;
   
   // If we already fetched Premium availability in the past seven days,
   // don't fetch it again.
   const checkingRemainingDays = Date.now() - 7 * 24 * 60 * 60 * 1000;
   if (typeof existingPremiumAvailability === "object" && 
-      typeof existingIntroPricingEndDate === "object" && 
-      existingPremiumAvailability.fetchedAt > checkingRemainingDays &&
-      existingIntroPricingEndDate.fetchedAt > checkingRemainingDays
+      existingPremiumAvailability.fetchedAt > checkingRemainingDays
```

Comment:
This may have a conflict now bc #414 landed (my bad)

Reply:
No worries, there was a bit of overlap too - rebased.